src/ngraph/frontend/onnx_import/op/quantize_linear.cpp

```diff
@@ -0,0 +1,99 @@
+//*****************************************************************************
+// Copyright 2017-2018 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include <cstddef>
+#include <cstdint>
+#include <iterator>
+#include <memory>
+
+#include "exceptions.hpp"
+#include "ngraph/axis_set.hpp"
+#include "ngraph/op/quantize.hpp"
+#include "ngraph/shape.hpp"
+#include "quantize_linear.hpp"
+
+namespace ngraph
+{
+    namespace onnx_import
+    {
+        namespace op
+        {
+            namespace set_1
+            {
+                ngraph::NodeVector quantize_linear(const ngraph::onnx_import::Node& node)
```

Comment:
We're already in the right namespace, there's no need to use fully qualified name.